core/src/main/kotlin/org/evomaster/core/output/service/TestWriterUtils.kt

```diff
@@ -0,0 +1,33 @@
+package org.evomaster.core.output.service
+
+import org.evomaster.core.EMConfig
+import org.evomaster.core.problem.external.service.httpws.HttpExternalServiceAction
+
+/**
+ * Helper functions for [TestCaseWriter] and [TestSuiteWriter]
+ */
+class TestWriterUtils {
+    companion object {
+        /**
+         * Check the current [EMConfig] and return boolean as the result
+         */
+        fun handleExternalService(config: EMConfig): Boolean {
+            if (config.externalServiceIPSelectionStrategy != EMConfig.ExternalServiceIPSelectionStrategy.NONE) {
```

Comment:
can simplify with simple: `return config.externalServiceIPSelectionStrategy != EMConfig.ExternalServiceIPSelectionStrategy.NONE` 
also, see previous comment about rename and move to `EMConfig`